framework/tlvf/yaml/tlvf/WSC/WSC_Attributes.yaml

```diff
@@ -680,3 +680,107 @@ cWscAttrOsVersion:
   os_version:
     _type: uint32_t
     _value: 0x80000001
+
+cWscAttrMac:
+  _type: class
+  _is_tlv_class: True
+  type: 
+    _type: eWscAttributes
+    _value: ATTR_MAC_ADDR
+  length:
+    _type: uint16_t
+    _value_const: WSC_MAC_LENGTH
```

Comment:
Why do we need const values inside the message?
It's part of the EasyMesh specification?

Reply:
A WSC attribute has a length and data, as a TLV does.
Specifying the length as `_value_const` is a tlvf feature which basically sets the `length()` getter to return a const so it can't be changed accidentally.